### PR TITLE
Added minor clarification about ipympl in docs

### PR DESCRIPTION
**Pull request by jlstevens**
Addresses comment made in https://github.com/holoviz/datashader/pull/1009.

### CODE REVIEW

File: examples/getting_started/3_Interactivity.ipynb [hunk 1]
```diff
@@ -298,7 +298,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package."
+    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package. If you do not have `ipympl` already, make sure to restart the notebook server after installation before re-executing the notebook."
```
Comment (jbednar) [inline examples/getting_started/3_Interactivity.ipynb:301]: Are you saying that if they already had ipympl installed but weren't yet using it, they don't need to restart the server and re-execute? I wouldn't have thought that installing would affect that.

Comment (jbednar) [reply examples/getting_started/3_Interactivity.ipynb:301]: Maybe: 

```suggestion
    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package, clear the notebook output, restart your notebook server, and re-execute the notebook."
```

Comment (jlstevens) [reply examples/getting_started/3_Interactivity.ipynb:301]: > Are you saying that if they already had ipympl installed but weren't yet using it, they don't need to restart the server and re-execute?

I'm not sure tbh. I think your suggestion is clearer anyway.

**Review by jbednar — APPROVED** [review]

**Comment by jlstevens** [comment]
I'll go ahead and merge this now.

**Pull request merged**